docs/modules/ROOT/pages/api/governance.adoc

```diff
@@ -1982,6 +1990,560 @@ Emits a {TimelockUpdated} event.
+* xref:#IMultisig-execute_transaction_batch[`++execute_transaction_batch(calls, salt)++`]
+--
+
+[.contract-index]
+.Events
+--
+* xref:#IMultisig-SignerAdded[`++SignerAdded(signer)++`]
+* xref:#IMultisig-SignerRemoved[`++SignerRemoved(signer)++`]
+* xref:#IMultisig-QuorumUpdated[`++QuorumUpdated(old_quorum, new_quorum)++`]
+* xref:#IMultisig-TransactionSubmitted[`++TransactionSubmitted(id, signer)++`]
+* xref:#IMultisig-TransactionConfirmed[`++TransactionConfirmed(id, signer)++`]
+* xref:#IMultisig-ConfirmationRevoked[`++ConfirmationRevoked(id, signer)++`]
+* xref:#IMultisig-TransactionExecuted[`++TransactionExecuted(id)++`]
+* xref:#IMultisig-CallSalt[`++CallSalt(id, salt)++`]
+--
+
+[#IMultisig-Functions]
+==== Functions
+
+[.contract-item]
+[[IMultisig-get_quorum]]
+==== `[.contract-item-name]#++get_quorum++#++() → u32++` [.item-kind]#external#
+
+Returns the current quorum value. The quorum is the minimum number of confirmations required to approve a transaction.
+
+[.contract-item]
+[[IMultisig-is_signer]]
+==== `[.contract-item-name]#++is_signer++#++(signer: ContractAddress) → bool++` [.item-kind]#external#
+
+Returns whether the given `signer` is registered. Only registered signers can submit, confirm, or execute transactions.
+
+[.contract-item]
+[[IMultisig-get_signers]]
+==== `[.contract-item-name]#++get_signers++#++() → Span<ContractAddress>++` [.item-kind]#external#
+
+Returns the list of all current signers.
+
+[.contract-item]
+[[IMultisig-is_confirmed]]
+==== `[.contract-item-name]#++is_confirmed++#++(id: TransactionID) → bool++` [.item-kind]#external#
+
+Returns whether the transaction with the given `id` has been confirmed.
+
+[.contract-item]
+[[IMultisig-is_confirmed_by]]
+==== `[.contract-item-name]#++is_confirmed_by++#++(id: TransactionID, signer: ContractAddress) → bool++` [.item-kind]#external#
+
+Returns whether the transaction with the given `id` has been confirmed by the specified `signer`.
+
+[.contract-item]
+[[IMultisig-is_executed]]
+==== `[.contract-item-name]#++is_executed++#++(id: TransactionID) → bool++` [.item-kind]#external#
+
+Returns whether the transaction with the given `id` has been executed.
+
+[.contract-item]
+[[IMultisig-get_transaction_state]]
+==== `[.contract-item-name]#++get_transaction_state++#++(id: TransactionID) → TransactionState++` [.item-kind]#external#
+
+Returns the current state of the transaction with the given `id`.
+
+[.contract-item]
+[[IMultisig-get_transaction_confirmations]]
+==== `[.contract-item-name]#++get_transaction_confirmations++#++(id: TransactionID) → u32++` [.item-kind]#external#
+
+Returns the number of confirmations from registered signers for the transaction with the specified `id`.
+
+[.contract-item]
+[[IMultisig-get_submitted_block]]
+==== `[.contract-item-name]#++get_submitted_block++#++(id: TransactionID) → u64++` [.item-kind]#external#
+
+Returns the block number when the transaction with the given `id` was submitted.
```

Comment:
Nit: `get_submitted_block` is listed before `get_transaction_state` in the interface. I'd move this up to follow the interface order. We should also adjust this in the component doc below as well as the actual function in the code so it's all nice and orderly

Reply:
Good catch, fixed the order of the functions